### ISSUE TITLE
"MaxListenersExceededWarning: Possible EventEmitter memory leak detected" when downloading

### ISSUE THREAD

**Issue by kjozwiak** [brave/browser-laptop#13565]
### Description

When downloading files using `0.22.6` e6ff4ea, you'll get the following error in the terminal/command prompt:

```
(node:74390) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 removed listeners added. Use emitter.setMaxListeners() to increase limit
MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 removed listeners added. Use emitter.setMaxListeners() to increase limit
    at _addListener (events.js:262:19)
    at DownloadItem.addListener (events.js:279:10)
    at DownloadItem.<anonymous> (/Applications/Brave-Beta.app/Contents/Resources/app.asar/app/filtering.js:596:12)
    at emitTwo (events.js:106:13)
    at DownloadItem.emit (events.js:194:7)
```

### Steps to Reproduce

  1. install/launch `0.22.6` e6ff4ea via the terminal/command prompt
  2. visit https://github.com/brave/browser-laptop/releases
  3. initiate a download

**Actual result:**

Brave displaying `node:74390) MaxListenersExceededWarning: Possible EventEmitter memory leak detected.` errors when downloading files.

**Expected result:**

Brave shouldn't be displaying `node:74390) MaxListenersExceededWarning: Possible EventEmitter memory leak detected.` errors when downloading files.

**Reproduces how often:**

100% reproducible using the above STR.

### Brave Version

**about:brave info:**

> Brave: 0.22.6 
> V8: 6.5.254.36 
> rev: e6ff4ea43ed740d1449a4c72005e22ce5235447e 
> libchromiumcontent: 65.0.3325.162
> Muon: 5.1.0 
> OS Release: 17.4.0 
> Update Channel: Beta 
> OS Architecture: x64 
> OS Platform: macOS 
> Node.js: 7.9.0 
> Brave Sync: v1.4.2 

**Reproducible on current live release:**

Yes, currently reproducible using `0.21.24` 19c78a1624118e344852c4c0f44c0d45be841b5f which is the latest released build.

### Additional Information

* macOS - reproduced by @LaurenWags and myself
* Win 10 x64 - reproduced by @srirambv 
* Ubuntu - reproduced by @btlechowski

**Comment by bsclifton** [comment]
Fixed in 0.23.x with https://github.com/brave/browser-laptop/pull/12909
cc: @petemill

**Comment by bsclifton** [comment]
Duplicate of https://github.com/brave/browser-laptop/issues/9643
